### DISCUSSION TITLE
Support for Amazfit GTR3 / GTR3PRO / GTS3

### DISCUSSION THREAD

**Discussion by Klaus3d3** [Navid200/xDrip#182] [Development]
Has anyone started with the developement of a watchface app for these watches?

Since they have a new OS called ZeppOS a lot of things seems to be possible.

I'm registered as an individual developer at HUAMIs developer program and have access to the documentation, some examples and Zepp Studio.

We now have BLECommunication with the companion app, clickable items on the watchface, eventlisteners and so on. Looks really promissing to hook on, at least, onto xDrips webservice easily and get a longlasting, functional smartwatch. 

But honestly i need some help with this Javascript shit.

**Comment by Navid200** [comment]
@Klaus3d3 Hi,

Discussions are now open in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

Would you please open a thread there and copy your post from this thread to it?
I have not closed these discussions because there are still a few threads here that need to be transferred.
But, please don't open any new threads here.

Please let me know if you have any questions.

Thanks
